conf/machine/include/vendor.inc

```diff
@@ -6,7 +6,7 @@ PROJECT_BRANCH ?= "${RDKE_GITHUB_BRANCH}"
 
 VENDOR_LAYER_EXTENSION = "${MACHINE}-vendor"
 
-VENDOR_IPK_SERVER_PATH = "${RDK_ARTIFACTS_BASE_URL}/opkg/${MACHINE}/ipks/${VENDOR_LAYER_EXTENSION}/4.0.0"
+VENDOR_IPK_SERVER_PATH = "https://partners.artifactory.comcast.com/artifactory/opkg/raspberrypi4-64-rdke/ipks/raspberrypi4-64-rdke-vendor/develop/rdk_gstreamerutils"
```

Comment:
Please confirm the IPK feed change is required or not.
You can generate a new IPK feed from vendor-dev (if there is any change there) and add that here.